gdsfactory/pdk.py

```diff
@@ -268,7 +268,13 @@ def get_component(self, component: ComponentSpec, **kwargs) -> Component:
                 if cell_name in self.cells
                 else self.containers[cell_name]
             )
-            return cell(**settings)
+            component = cell(**settings)
+            component = (
+                self.default_decorator(component) or component
+                if self.default_decorator
+                else component
+            )
+            return component
```

Comment:
@joamatab what's the purpose of this change?
we already [apply the active PDK's default decorator when instantiating the component](https://github.com/gdsfactory/gdsfactory/blob/83bb9359bfbca9b08f61874693d523676dc82ead/gdsfactory/cell.py#L127) through the `@cell` decorator (if it is not explicitly overridden when calling the component function). wouldn't this double-decorate the component? and only do so when called via `get_component()`?